web/packages/teleport/src/Kubes/ConnectDialog/ConnectDialog.tsx

```diff
@@ -99,6 +133,64 @@ function ConnectDialog(props: Props) {
             <TextSelectCopy mt="2" text={`tsh request drop`} />
           </Box>
         )}
+        <Validation>
+          {() => (
+            <Box borderTop={1} mb={4} mt={4}>
+              <Text mt={3} bold>
+                Or exec into a pod on this Kubernetes cluster
+              </Text>
+              <Flex gap={3}>
+                <FieldInput
+                  value={execPath}
+                  placeholder="namespace/pod"
+                  label="Pod to exec into"
+                  width="50%"
+                  onChange={e => setExecPath(e.target.value.trim())}
+                  toolTipContent={
+                    <Text>
+                      Specify namespace and pod you want to exec into.
+                      Optionally you can also specify container by adding it at
+                      the end: '/namespace/pod/container'
+                    </Text>
+                  }
+                />
+                <FieldInput
+                  rule={requiredField('Command to execute is required')}
+                  value={execCommand}
+                  placeholder="/bin/bash"
+                  label="Command to execute"
+                  width="50%"
+                  onChange={e => setExecCommand(e.target.value)}
+                  toolTipContent={
+                    <Text>
+                      The command that will be executed inside the target pod.
+                    </Text>
+                  }
+                />
+              </Flex>
+              <Flex justifyContent="space-between" gap={3}>
+                <Toggle
+                  isToggled={execInteractive}
+                  onToggle={() => {
+                    setExecInteractive(b => !b);
+                  }}
+                >
+                  <Box ml={2} mr={1}>
+                    Interactive shell
+                  </Box>
+                  <ToolTipInfo>
+                    You can start an interactive shell and have a bidirectional
+                    communication with the target pod, or you can run one-off
+                    command and see its output.
+                  </ToolTipInfo>
+                </Toggle>
+                <ButtonPrimary onClick={startKubeExecSession}>
+                  Run Command
+                </ButtonPrimary>
```

Comment:
If we wrap the form into `<form>`, add `onSubmit` on it and change the button's type to `submit` and then add autofocus on the first field, then it'll be possible to navigate the form using keyboard and submit it with Enter.

<details>
<summary>Patch</summary>

Copy then `pbpaste | git apply`

```diff
diff --git a/web/packages/teleport/src/Kubes/ConnectDialog/ConnectDialog.tsx b/web/packages/teleport/src/Kubes/ConnectDialog/ConnectDialog.tsx
index eb06d90ba8..ed7e858f08 100644
--- a/web/packages/teleport/src/Kubes/ConnectDialog/ConnectDialog.tsx
+++ b/web/packages/teleport/src/Kubes/ConnectDialog/ConnectDialog.tsx
@@ -56,7 +56,8 @@ function ConnectDialog(props: Props) {
   const [execCommand, setExecCommand] = useState('');
   const [execInteractive, setExecInteractive] = useState(true);
 
-  const startKubeExecSession = () => {
+  const startKubeExecSession = event => {
+    event.preventDefault();
     const splitPath = execPath.split('/');
 
     const url = cfg.getKubeExecConnectRoute(
@@ -136,58 +137,59 @@ function ConnectDialog(props: Props) {
         <Validation>
           {() => (
             <Box borderTop={1} mb={4} mt={4}>
-              <Text mt={3} bold>
-                Or exec into a pod on this Kubernetes cluster
-              </Text>
-              <Flex gap={3}>
-                <FieldInput
-                  value={execPath}
-                  placeholder="namespace/pod"
-                  label="Pod to exec into"
-                  width="50%"
-                  onChange={e => setExecPath(e.target.value.trim())}
-                  toolTipContent={
-                    <Text>
-                      Specify namespace and pod you want to exec into.
-                      Optionally you can also specify container by adding it at
-                      the end: '/namespace/pod/container'
-                    </Text>
-                  }
-                />
-                <FieldInput
-                  rule={requiredField('Command to execute is required')}
-                  value={execCommand}
-                  placeholder="/bin/bash"
-                  label="Command to execute"
-                  width="50%"
-                  onChange={e => setExecCommand(e.target.value)}
-                  toolTipContent={
-                    <Text>
-                      The command that will be executed inside the target pod.
-                    </Text>
-                  }
-                />
-              </Flex>
-              <Flex justifyContent="space-between" gap={3}>
-                <Toggle
-                  isToggled={execInteractive}
-                  onToggle={() => {
-                    setExecInteractive(b => !b);
-                  }}
-                >
-                  <Box ml={2} mr={1}>
-                    Interactive shell
-                  </Box>
-                  <ToolTipInfo>
-                    You can start an interactive shell and have a bidirectional
-                    communication with the target pod, or you can run one-off
-                    command and see its output.
-                  </ToolTipInfo>
-                </Toggle>
-                <ButtonPrimary onClick={startKubeExecSession}>
-                  Run Command
-                </ButtonPrimary>
-              </Flex>
+              <form onSubmit={startKubeExecSession}>
+                <Text mt={3} bold>
+                  Or exec into a pod on this Kubernetes cluster
+                </Text>
+                <Flex gap={3}>
+                  <FieldInput
+                    value={execPath}
+                    placeholder="namespace/pod"
+                    autoFocus
+                    label="Pod to exec into"
+                    width="50%"
+                    onChange={e => setExecPath(e.target.value.trim())}
+                    toolTipContent={
+                      <Text>
+                        Specify namespace and pod you want to exec into.
+                        Optionally you can also specify container by adding it
+                        at the end: '/namespace/pod/container'
+                      </Text>
+                    }
+                  />
+                  <FieldInput
+                    rule={requiredField('Command to execute is required')}
+                    value={execCommand}
+                    placeholder="/bin/bash"
+                    label="Command to execute"
+                    width="50%"
+                    onChange={e => setExecCommand(e.target.value)}
+                    toolTipContent={
+                      <Text>
+                        The command that will be executed inside the target pod.
+                      </Text>
+                    }
+                  />
+                </Flex>
+                <Flex justifyContent="space-between" gap={3}>
+                  <Toggle
+                    isToggled={execInteractive}
+                    onToggle={() => {
+                      setExecInteractive(b => !b);
+                    }}
+                  >
+                    <Box ml={2} mr={1}>
+                      Interactive shell
+                    </Box>
+                    <ToolTipInfo>
+                      You can start an interactive shell and have a
+                      bidirectional communication with the target pod, or you
+                      can run one-off command and see its output.
+                    </ToolTipInfo>
+                  </Toggle>
+                  <ButtonPrimary type="submit">Run Command</ButtonPrimary>
+                </Flex>
+              </form>
             </Box>
           )}
         </Validation>

```
</details>